config/metac.yaml

```diff
@@ -11,6 +11,56 @@ spec:
   attachments:
   - apiVersion: dao.mayadata.io/v1alpha1
     resource: openebses
+  - apiVersion: apps/v1
```

Comment:
can we use a better name instead of current namespace i.e. openebs-test?
how about openebs-operator